### PR TITLE
Gen 9 Follower pokemon sprites

### PR DESCRIPTION
**Pull request by Liamjd14**
Adds most of the gen 9 pokemon ow / follower sprites. 

## Description

Adds images and data for all gen 9 pokemon to have follower sprites except for terapagos stellar form and roaming ghimmigoul. 
I plan on adding asym sprites for scovillain, and the tinkaton line in the future (the sprites are allrady there, just the code needs adding to get them to work).

## Images

Screenshots of some of the follower sprites working:

pokeemerald-0
pokeemerald-1
pokeemerald-2
pokeemerald-3
pokeemerald-4
pokeemerald-5
pokeemerald-6
pokeemerald-7
pokeemerald-8

## **People who collaborated with me in this PR**

Formatted by: Liamjd14
Original sprites by:
Darkus_Shadow, Princess-Phoenix, shaderr31, Molfang62, CarmaNekko, EduarPokeN, Larryturbo, TyranitarDark, Anarlaurendil
Sounces: 
Normal: https://www.deviantart.com/darkusshadow/art/Gen-9-Paldea-Pokemon-Overworld-Sprites-967776690
Shiny: https://www.deviantart.com/darkusshadow/art/SHINY-Gen-9-Paldea-Pokemon-Overworld-Sprites-967779547

## **Discord contact info**

Liamjd14

### CODE REVIEW

File: graphics/pokemon/ogerpon/hearthflame/overworld.png
Comment (hedara90) [inline graphics/pokemon/ogerpon/hearthflame/overworld.png:1]: Wrong color for the red part of mask and mantle.

File: graphics/pokemon/archaludon/overworld.png
Comment (hedara90) [inline graphics/pokemon/archaludon/overworld.png:1]: Blinking pixels on east/west body in front of arms.

File: graphics/pokemon/gouging_fire/overworld.png
Comment (hedara90) [inline graphics/pokemon/gouging_fire/overworld.png:1]: Normal palette, red part of crown missing on south sprite. Claws should be red.
Shiny crown too brown.

File: graphics/pokemon/raging_bolt/overworld.png
Comment (hedara90) [inline graphics/pokemon/raging_bolt/overworld.png:1]: Shiny body wrong color.

File: graphics/pokemon/iron_boulder/overworld.png
Comment (hedara90) [inline graphics/pokemon/iron_boulder/overworld.png:1]: Wrong eye color on both palettes.

File: graphics/pokemon/terapagos/terastal/overworld.png
Comment (hedara90) [inline graphics/pokemon/terapagos/terastal/overworld.png:1]: Bottom outline on shiny blinking.
Nose looks cut off.
Color of fur/body/stuff is too grey.
Top jewel is wrong colors.
Missing a lot of shading.

File: src/data/pokemon/species_info/gen_5_families.h [hunk 1]
```diff
@@ -11950,4 +11958,4 @@ const struct SpeciesInfo gSpeciesInfoGen5[] =
 
 #ifdef __INTELLISENSE__
 };
-#endif
+#endif
```
Comment (hedara90) [inline src/data/pokemon/species_info/gen_5_families.h:11961]: Fix EOL

**Comment by Liamjd14** [comment]
Hi there, I have most of the changes done on my end. A couple of them i have been unable to add due to colour pallet restrictions.
The terapagos sprite now has the right colors but currently the  shiny dosen't have the right head color due to there not beign any free space on the pallet.

Also whats with the last 'fix EOL' as i have checked the files on my end and the shown change on here is allready there?

**Comment by hedara90** [comment]
> Also whats with the last 'fix EOL' as i have checked the files on my end and the shown change on here is allready there?

20240904_21h22m14s_grim
It's a thing where the last line in the file doesn't have a `\n`.
It's a thing that some text editors do, text editors should warn you if the file you're editing doesn't have one.

>I have most of the changes done on my end. A couple of them i have been unable to add due to colour pallet restrictions.
The terapagos sprite now has the right colors but currently the shiny doesn't have the right head color due to there not beign any free space on the pallet.

Mark the ones that are fixed as resolved.
We can deal with ones that require more involved fixes later.

**Comment by Liamjd14** [comment]
I have marked off all the ones im sure are fixed. just left the ones that might need more work.
Klawf - not sure what is needed to make it look better
Iron juggulis - not sure on how to improve the north sprite 
Gouging fire - not sure if the colors i have used are right or not
Terapagos - still needs some work as shiny is currently inaccurate due to pallet limitations

also i coppied the change on the 'fix EOL' to the file. hopefully it saved but might be worth checking as im not sure

**Comment by hedara90** [comment]
The EOL was fixed, I marked it as resolved when I saw it change.
Gouging Fire looks good now.
Iron Juggulis I don't actually know what can be done with.

**Comment by Liamjd14** [comment]
Thats good then, if you think gouging fire is good then i will mark it off. Didnt know what to do with iron juggulis myself due to its color scheme.

**Comment by Liamjd14** [comment]
What do you think to these two. Updated Klawf Shiny, and Terapagos rework (slightly change colors and shiny.)

Terapagos OW:
moverworld
mmoverworld

Klawf OW:
overworld

**Comment by Liamjd14** [comment]
Hi there, I have pushed the changes to it and resolved the relevant changes.

File: graphics/pokemon/frigibax/overworld.png
Comment (hedara90) [inline graphics/pokemon/frigibax/overworld.png:1]: Shiny hand things should be pink, not yellow.

File: graphics/pokemon/gouging_fire/overworld.png
Comment (hedara90) [inline graphics/pokemon/gouging_fire/overworld.png:1]: Normal palette sideburns should be red, not brown.

File: graphics/pokemon/great_tusk/overworld.png
Comment (hedara90) [inline graphics/pokemon/great_tusk/overworld.png:1]: Shiny pal has wrong colors on tusks on south, but might be hard to solve.

File: graphics/pokemon/houndstone/overworld.png
Comment (hedara90) [inline graphics/pokemon/houndstone/overworld.png:1]: Two brown pixels under tombstone, are they supposed to be there?
I don't really like the south sprite's mouth part, it looks like eyes where it shouldn't be eyes.

File: graphics/pokemon/klawf/overworld.png
Comment (hedara90) [inline graphics/pokemon/klawf/overworld.png:1]: Shiny cheeks wrong color, there's enough room in both palettes for a new color for the cheeks.

File: graphics/pokemon/koraidon/overworld.png
Comment (hedara90) [inline graphics/pokemon/koraidon/overworld.png:1]: Blinking north sprite right side back feathers

File: graphics/pokemon/lokix/overworld.png
Comment (hedara90) [inline graphics/pokemon/lokix/overworld.png:1]: Shiny face outline blinking pixels

File: graphics/pokemon/ogerpon/hearthflame/overworld.png
Comment (hedara90) [inline graphics/pokemon/ogerpon/hearthflame/overworld.png:1]: Shiny palette still incorrect.

File: graphics/pokemon/oinkologne/female/overworld.png
Comment (hedara90) [inline graphics/pokemon/oinkologne/female/overworld.png:1]: Shiny top of ear blinking on east/west.

File: graphics/pokemon/quaquaval/overworld.png
Comment (hedara90) [inline graphics/pokemon/quaquaval/overworld.png:1]: North sprite has blinking pixel on the orange eyebrow thing.

File: graphics/pokemon/tandemaus/overworld.png
Comment (hedara90) [inline graphics/pokemon/tandemaus/overworld.png:1]: Stray pixel on south sprite by left ear in transparent area.

File: graphics/pokemon/tinkaton/overworld.png
Comment (hedara90) [inline graphics/pokemon/tinkaton/overworld.png:1]: Blinking pixel on east/west sprite by the hair tuft, more visible on shiny palette.

File: graphics/pokemon/wo_chien/overworld.png
Comment (hedara90) [inline graphics/pokemon/wo_chien/overworld.png:1]: Single blinking pixel on north sprite on the right half of the sprite, upper part of the lowest spiral.

File: graphics/pokemon/wooper/wooper_paldean/overworld.png
Comment (hedara90) [inline graphics/pokemon/wooper/wooper_paldean/overworld.png:1]: Already had a Paldean Wooper.

File: src/data/pokemon/species_info/gen_2_families.h [hunk 2]
```diff
@@ -3073,7 +3073,7 @@ const struct SpeciesInfo gSpeciesInfoGen2[] =
             sPicTable_Clodsire,
             SIZE_32x32,
             SHADOW_SIZE_M,
-            TRACKS_FOOT,
+            TRACKS_SLITHER,
```
Comment (hedara90) [inline src/data/pokemon/species_info/gen_2_families.h:3076]: Already had a Clodsire

**Review by hedara90 — COMMENTED** [review]
Another bunch of things noticed.

File: graphics/pokemon/great_tusk/overworld.png
Comment (Liamjd14) [reply graphics/pokemon/great_tusk/overworld.png:1]: Resolved by swaping color to other grey

File: graphics/pokemon/dondozo/overworld.png
[the remark this answers is not in the document]
Comment (Liamjd14) [reply graphics/pokemon/dondozo/overworld.png:1]: resolved by combining similar colors in to one shade

**Comment by Liamjd14** [comment]
all changes show on my end.

Fixes:
Blinking Ear on dachbun
Dondozo - change pallet to keep white eye on shiny
Frigibax - change pallet to show pink hands on shiny
gouging fire - change pallet to allow for red on normal and aditional fix to shiny
great tusk - simplified grey to allow for right color on tusks
klawf - change pallet to show correct cheeks
koraidon - fixed blinking
lokix - fixed shiny blinking
ogerpon - change color to reflect shiny battle sprite
oinkalone - fixes blinking
quaquavel - fixed blinking colors
tandamous - removed stray pixel
tinkaton - fixed blinking pixel
wochen - fixed blinking pixel
houndstone - fixed brown pixel on left-right sprite

how is best to handle the wooper and clodsire sprites as i realize there were already ones? should i just delete the files on mine or replace them with the ones from the expansion repo.

**Comment by hedara90** [comment]
You should be able to add back the ones that were already there, worst you can copy the sprite and palettes back in from `master`.

**Comment by Liamjd14** [comment]
That should be all of the changes done then.

**Review by hedara90 — APPROVED** [review]

**Comment by hedara90** [comment]
Awesome work getting all these done!

**Comment by Liamjd14** [comment]
Just glad i can contribute to the expansion. Will have a look in the sprite issues when i get a second and see if there are i can help with.

**Pull request merged**